astropy/time/tests/test_delta.py

```diff
@@ -605,3 +608,42 @@ def test_insert_timedelta():
     # Insert a scalar using an auto-parsed string
     tm2 = tm.insert(1, TimeDelta([10, 20], format='sec'))
     assert np.all(tm2 == TimeDelta([1, 10, 20, 2], format='sec'))
+
+
+@pytest.mark.filterwarnings("error")
```

Comment:
This should not be needed, as it is the default for all astropy tests!

Reply:
I wanted to be explicit here since the test is about warnings.

Reply:
Makes sense in principle, but since we do not do this anywhere else, I think it is fine to just omit it here too.

Reply:
ok, done